app/templates/components/render-item.hbs

```diff
@@ -3,10 +3,12 @@
     <span class="list__cell-arrow"></span>
     <span title="{{model.name}}">
       <span class="js-render-profile-name">{{model.name}}</span>
-      <span class="pill pill_not-clickable js-render-profile-duration">{{ms-to-time model.duration}}</span>
     </span>
   {{/list.cell}}
   {{#list.cell class="list__cell_value_numeric js-render-profile-timestamp"}}
```

Comment:
`js-render-profile-timestamp` does not apply to this column anymore. Maybe `js-render-profile-duration`.